bot/cogs/free.py

```diff
@@ -80,7 +79,8 @@ class Free(Cog):
             # Sort channels in descending order by seconds
             # Get position in list, inactivity, and channel object
             # For each channel, add to embed.description
-            for i, (inactive, channel) in enumerate(sorted(free_channels, reverse=True), 1):
+            for i, (inactive, channel) in enumerate(
+                    sorted(free_channels, key=lambda free_channel: free_channel[0], reverse=True), 1):
```

Comment:
This is getting a bit messy now with the key. At this point it'd be cleaner to assign the output of `sorted()` to a variable on a separate line and then use the variable inside `enumerate()`.